music_metadata_extractor/resolver.py

```diff
@@ -23,18 +26,35 @@ def _is_youtube(input_url: str) -> bool:
     return False
 
 
+def _is_youtube_music(input_url: str) -> bool:
+    if bool(
+        re.match(
+            r"http(?:s?):\/\/(?:(www)\.)?music.youtube.com*",
+            input_url,
+        )
+    ):
+        return True
+    return False
+
+
 def _resolve_link(input_url: str) -> SupportedSources:
     """Resolve the input link platform"""
     if _is_youtube(input_url):
         return SupportedSources.youtube
-    raise ValueError("Unsupported URL!")
+    elif _is_youtube_music(input_url):
+        return SupportedSources.youtube_music
+    else:
+        raise ValueError("Unsupported URL!")
 
 
 def get_source_data(input_url: str) -> Tuple[BaseProviderInput, dict]:
     """Get input object required by providers to fetch metadata"""
     source_type = _resolve_link(input_url)
     if source_type == SupportedSources.youtube:
         provider_input, extraAttrs = get_yt_info(input_url)
+    elif source_type == SupportedSources.youtube_music:
+        provider_input, extraAttrs = get_yt_info(input_url)
```

Comment:
`get_yt_info` takes in a YouTube link. You'll need to convert it first using `yt_music_link_to_yt_link` and then pass it to `get_yt_info`